Makefile

```diff
@@ -0,0 +1,29 @@
+all: build
+	@echo "Built"
+
+start: build
+	(cd ./tests; ./scheduled > scheduled.log 2>&1 &)
+	(cd ./tests; tail -f *.log)
+
+stop:
+	-pkill scheduled
+
+clean: stop
+	rm -rf ./tests/data
+	rm -f ./tests/*.log
+	rm -f ./tests/*.key
+
+fulltest: build
+	(cd ./tests; ./simple_test.sh)
+	tail ./tests/*.log
+
+build:
+	go build -o tests/register cmd/register.go
+	go build -o tests/scheduled cmd/scheduled.go
+	go build -o tests/schedule cmd/schedule.go 
+
+install:
+	@echo "Install into root"
+
+test:
+	go test -v github.com/phomer/scheduler/datastore
```

Comment:
Most of those targets should be marked as phony

Reply:
Yes, I think all of them are currently phony, but it got me wondering if I could restructure the build targets to be driven from their file names?

Reply:
There is probably some way you could do that, but don't get hung up on the makefile 🙂 the current targets are pretty standard and well known, I would just declare them as phony so files or folders are not interfering with them